src/vdaf/prio3.rs

```diff
@@ -109,7 +113,7 @@ impl<F: FieldElement> TryFrom<Vec<F>> for Prio3Result<u64> {
 
 /// Aggregate result for vector data types.
 #[derive(PartialEq, Eq)]
-pub struct Prio3ResultVec<T: Eq>(Vec<T>);
+pub struct Prio3ResultVec<T: Eq>(pub Vec<T>);
```

Comment:
Instead of making the anonymous field `pub`, can we make this type "look" like a vector? Like, is there a Rusty way to overload `res[i]` and `res.len()`, etc.?

Reply:
I can no longer recall why I ever needed that to be `pub`, but it looks like I don't anymore, so I undid this change.